frontends/infinite-corridor/package.json

```diff
@@ -35,14 +36,17 @@
     "ol-util": "workspace:*",
     "ol-widgets": "workspace:*",
     "postcss-loader": "^7.0.1",
-    "react": "^16.14",
-    "react-dom": "^16.14",
+    "react": "^18.2",
+    "react-dom": "^18.2",
+    "react-infinite-scroller": "^1.2.6",
     "react-router": "^5.3.3",
     "react-router-dom": "^5.1.2",
-    "rmwc": "5.7.1",
+    "sass": "^1.62.1",
+    "sass-loader": "^13.3.0",
```

Comment:
`rmwc` was a mistake and is not used in infinite-corridor.

`sass`, `sass-loader`, and the other "new" dependencies, are not really new: we were using them before, but they were not explicitly listed by infinite-corridor. (Workspace `open-discussions` was using them, they got hoisted to root, which enabled their use in infinite-corridor. This PR stops hoisting of open-discussions's dependencies, so now they *need* to be explicitly listed.